src/System.CodeDom/tests/System.CodeDom.Tests.csproj

```diff
@@ -0,0 +1,41 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
+    <OutputType>Library</OutputType>
+    <!--
+      Until we get first class support for NS1.7 and Netcoreapp1.1
```

Comment:
I don't expect that this hard-coding of TestTFM netcoreapp1.1 is needed any longer.